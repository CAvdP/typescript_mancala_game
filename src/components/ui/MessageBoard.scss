.message-board {
    border-radius: 10px;
    height: 160px;
    width: 800px;
    margin: 40px;
    font-size: 6rem;
    display: flex;
    align-items: center;
    justify-content: center;
    color: white;
}