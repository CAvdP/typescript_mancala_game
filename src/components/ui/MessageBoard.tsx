import React from 'react';
import './MessageBoard.scss';

interface Props {
 playerOneTurn: boolean;
}

function MessageBoard({playerOneTurn}: Props){
    const message = playerOneTurn? 'Player one': 'Player two';

    return (
        <button className={'message-board neon_pink_blue'} onClick={()=>{}}>{message}</button>
    )
}

export default MessageBoard