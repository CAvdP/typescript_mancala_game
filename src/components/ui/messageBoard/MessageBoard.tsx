import React from 'react';
import './styling/MessageBoard.scss';

interface Props {
 playerOneTurn: boolean;
}

function MessageBoard({playerOneTurn}: Props){
    const message = playerOneTurn? 'Player one': 'Player two';

    return (
        <button className={'message-board neon_pink_blue'} onClick={()=>{}}>{message}</button>
    )
}

export default MessageBoard