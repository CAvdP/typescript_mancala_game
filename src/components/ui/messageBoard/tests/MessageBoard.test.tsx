import React from 'react'
import renderer from 'react-test-renderer'
import MessageBoard from "../MessageBoard"



describe("MessageBoard Component", () => {
    it('matches the saved snapshot', () => {
        const actualSnapshot = renderer
          .create(<MessageBoard playerOneTurn={false} />)
          .toJSON()
          
        expect(actualSnapshot).toMatchSnapshot()
      });
});
