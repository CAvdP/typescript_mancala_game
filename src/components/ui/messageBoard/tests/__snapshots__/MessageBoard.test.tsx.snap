// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageBoard Component matches the saved snapshot 1`] = `
<button
  className="message-board neon_pink_blue"
  onClick={[Function]}
>
  Player two
</button>
`;
